styles.css

```diff
@@ -0,0 +1,90 @@
+/* .heading {
+  background-color: #b7e9f7;
+} */
+body {
+  background: #b7e9f7;
+}
+
+.app-wrap {
+  display: flex;
+  flex-direction: column;
+  max-width: 375px;
+  height: 130vh;
+  margin: 5vh auto 5vh auto;
+  border-radius: 15px;
+  background: #fff;
+}
+
+.app-header h1 {
+  width: 90px;
+  height: 28px;
+  padding-left: 5%;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  color: #28352f;
+}
+
+.app-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.primary h4 {
+  display: flex;
+  justify-content: space-around;
+  flex-direction: column;
+  width: 327px;
+  margin: 10px 30px 0 30px;
+  font-size: 35px;
+  font-style: normal;
+  font-weight: 700;
+  letter-spacing: 0;
+  text-align: center;
+}
+
+.primary h4:first-child {
+  margin-top: 58px;
+}
+
+.primary h4:last-child {
+  color: #3cb371;
+}
+
+.social-media {
+  display: flex;
+  justify-content: center;
+}
```

Comment:
- [ ] Right now, all your icons are being displayed in a blue color. To match the original design better, you should consider making them `#505F79`. To help you style them, you can check out [this page](https://developer.mozilla.org/en-US/docs/Web/CSS/:link).
- [ ] The icon sizes are also a bit different, consider double-checking the size in Figma.

Figma design:
![image](https://user-images.githubusercontent.com/66967274/148312920-6a747876-9aa6-4d3a-8427-65314f1b1cc1.png)


Your project:
![image](https://user-images.githubusercontent.com/66967274/148312871-dbee8845-7428-47f1-955d-88657e7d3212.png)